actions/workflows/st2workroom_st2enterprise_test.yaml

```diff
@@ -56,9 +56,24 @@
           DISABLE_HUBOT: "true"
           CI: "true"
         hosts: "{{hostname}}"
-        cmd: "curl -sSL https://stackstorm.com/install.sh -o /tmp/bootstrap.sh && chmod +x /tmp/bootstrap.sh && /tmp/bootstrap.sh -a /tmp/answers.yaml"
```

Comment:
I do not understand either but for whatever reason it did not fail. This does so I stopped fighting :)